src/trading/ClaimProceeds.sol

```diff
@@ -0,0 +1,51 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+import 'ROOT/reporting/Interfaces.sol';
+import 'ROOT/extensions/MarketFeeCalculator.sol';
+
+/**
+ * @title ClaimProceeds
+ * @dev This allows users to claim their money from a market by exchanging their shares 
+ */
+contract ClaimProceeds is Controlled {
+	uint256 constant THREE_DAYS = 3 * 24 * 60 * 60;
+
+	function publicClaimProceeds(IMarket market) onlyInGoodTimes returns(uint256) {
```

Comment:
Never mind about making it public.  I saw Micah's comment on Slack about making it internal instead.